affiliates/settings/base.py

```diff
@@ -430,3 +430,7 @@ def facebook_app_url_lazy():
 
 # List Of Locales Where Firefox Friends Messaging Is Translated & Enabled
 FRIENDS_MSG_LOCALES = ['cs', 'de', 'en-US', 'fr', 'nl', 'pt-BR', 'sl', 'sr']
+
+# Limit to three counts per minute.
+CLICK_THROTTLE_TIMEOUT = 60  # One minute
+MAX_CLICK_COUNT = 3
```

Comment:
With entire offices or companies sharing a single external IP, only allowing 3 clicks per minute per IP seems a bit restrictive. Not sure how many the system can handle, but I'm sure 10 or even 30 per min would be okay.